src/components/replica-status/index.js

```diff
@@ -1,10 +1,15 @@
+import { run } from '@equinor/eds-icons';
 import React from 'react';
 
 import { StatusBadge } from '../status-badge';
 
 export const ReplicaStatus = ({ replica }) => {
-  let status = replica ? replica.status : 'warning';
-  return <StatusBadge type={status}>{status}</StatusBadge>;
+  const status = replica ? replica.status : 'warning';
+  return (
+    <StatusBadge type={status} customIconData={status === 'Running' && run}>
+      {status}
+    </StatusBadge>
```

Comment:
Only pass `customIconData` when the status is `Running`
```suggestion
    <StatusBadge
      type={status}
      {...(status === 'Running' && { customIconData: run })}
    >
      {status}
    </StatusBadge>
```